model/cicd/resources.yaml

```diff
@@ -0,0 +1,31 @@
+groups:
+  - id: resource.cicd.pipeline
+    type: resource
+    stability: development
+    name: cicd.pipeline
+    brief: >
+        A pipeline is a series of automated steps that helps software teams deliver code.
+    attributes:
+      - ref: cicd.pipeline.name
+
+  - id: resource.cicd.pipelinerun
+    type: resource
+    stability: development
+    name: cicd.pipelinerun
```

Comment:
```suggestion
    name: cicd.pipeline.run
```

Reply:
Renamed in 77cca2d9657f7c4b02bfafdc6c444be5aafd7a30